invenio_drafts_resources/records/api.py

```diff
@@ -22,6 +22,7 @@
 """
 
 import uuid
+from datetime import datetime, timedelta
```

Comment:
moderate: is `timedelta` used?
```suggestion
from datetime import datetime
```